openwisp_firmware_upgrader/migrations/0003_default_permissions.py

```diff
@@ -0,0 +1,52 @@
+from django.db import migrations
+from django.contrib.auth.models import Permission
+
+from . import create_default_permissions
+
+
+
+def assign_permissions_to_groups(apps, schema_editor):
+    create_default_permissions(apps, schema_editor)
+    Group = apps.get_model('openwisp_users', 'Group')
+
+    try:
+        admin = Group.objects.get(name='Administrator')
+        operator = Group.objects.get(name='Operator')
+    # consider failures custom cases
+    # that do not have to be dealt with
+    except Group.DoesNotExist:
+        return
+
+    operators_read_only_admins_manage = ['build', 'devicefirmware', 'firmwareimage', 'batchupgradeoperation', 'upgradeoperation']
+    admins_can_manage = ['category']
+    manage_operations = ['add', 'change', 'delete']
+
+    for action in manage_operations:
+        for model_name in admins_can_manage:
+            permission = Permission.objects.get(
+                codename='{}_{}'.format(action, model_name)
+            )
+            admin.permissions.add(permission.pk)
+    for model_name in operators_read_only_admins_manage:
+        try:
+            permission = Permission.objects.get(codename='view_{}'.format(model_name))
+            operator.permissions.add(permission.pk)
+        except Permission.DoesNotExist:
+            pass
+        for action in manage_operations:
+            permission_ad = Permission.objects.get(codename='{}_{}'.format(action, model_name))
+            admin.permissions.add(permission_ad.pk)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('firmware_upgrader', '0001_initial'),
```

Comment:
this is migration number 0003 and is pointing to 0001, that's because 0002 is the private storage field.
Is this ok? Or should the order be changed depending on which one will be merged first?

Reply:
We should probably fix it at https://github.com/openwisp/openwisp-radius/blob/master/openwisp_radius/migrations/0004_default_permissions.py too